libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -453,3 +454,16 @@ Scope& AsmAnalyzer::scope(Block const* _block)
 	solAssert(scopePtr, "Scope requested but not present.");
 	return *scopePtr;
 }
+
+void AsmAnalyzer::expectValidType(string const& type, SourceLocation const& _location)
+{
+	if (!m_julia)
+		return;
+
+	if (!(set<string>{"bool", "u8", "s8", "u32", "s32", "u64", "s64", "u128", "s128", "u256", "s256"}).count(type))
+		m_errors.push_back(make_shared<Error>(
+			Error::Type::TypeError,
+			"User defined types (\"" + type + "\") are not supported yet.",
```

Comment:
Perhaps better `<type> is not a valid type (user defined types are not yet supported)`?